src/Docker/linux/centos/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM centos:7
+LABEL maintainers="GitTools Maintainers"
+
+ENV DOTNET_VERSION='2.2' 
```

Comment:
Could you set the version to 2.1 to have the same version as for other docker images? I'll make some changes later to add support for building both 2.1 and 2.2 versions